src/loaders/compute_tools/tool_common.py

```diff
@@ -584,6 +595,46 @@ def find_gtdbtk_summary_files(output_dir: Path):
     return summary_files
 
 
+def get_filtered_or_failed_genome_ids(output_dir: Path):
+    genome_ids = list()
+
+    # process filtered.tsv files
+    align_dir = output_dir / "align"
+    filter_files = [file_name for file_name in os.listdir(align_dir) if 
+                    re.search(loader_common_names.GTDB_FILTER_FILE_PATTERN, file_name)]
+    if not filter_files or len(filter_files) > 2:
+        raise ValueError("Parsed unexpected filterd.tsv files {filter_files}.")
```

Comment:
Can you be more specific about why the files were unexpected in the error message?

Reply:
Updated!

Reply:
:+1: